devstats/ghadb_postgres.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ghadb
+  namespace: devstats
+  labels:
+    app:  postgre
+spec:
+  ports:
+  - port: 5432
+  clusterIP: None
+  selector:
+    app: postgre
+
+---
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: postgre
+  namespace: devstats
+spec:
+  serviceName: ghadb
+  replicas: 1
+  selector:
+    matchLabels:
+      app: postgre
+  template:
+    metadata:
+      labels:
+        app: postgre
+    spec:
+      containers:
+      - name: postgres
+        image: postgres:10.1-alpine
+        env:
+        # For a list of POSTGRES environment variables see
+        # https://hub.docker.com/_/postgres/
+        - name: POSTGRES_USER
+          value: gha_admin
+        - name: POSTGRES_PASSWORD
+          value: password
+        - name: POSTGRES_DB
+          value: gha
```

Comment:
Probably logs database also needed here (or maybe logs database will be in a different pod/yaml/deployment)?